source/patterns/04-pages/research-article/research-article~en.yaml

```diff
@@ -102,6 +102,9 @@ contentHeader:
       - content:
           text: Nanjing Normal University, China
 
+contextualData:
+  doi: 10.5468/ogs.2016.59.1.1
```

Comment:
Since the content here is based on a real article, should probably use the actual DOI (10.7554/eLife.38827).